src/itest/kotlin/nl/lifely/zac/itest/NotificationsTest.kt

```diff
@@ -162,17 +163,17 @@ class NotificationsTest : BehaviorSpec({
                       "identificatie" : "999992958",
                       "roltoelichting" : "Overgenomen vanuit de product aanvraag",
                       "roltype" : "Bewindvoerder",
-                      "type" : "NATUURLIJK_PERSOON"
+                      "type" : "${BetrokkeneType.NATUURLIJK_PERSOON}"
```

Comment:
for some reason the `type` changed from "NATUURLIJK_PERSOON" _(uppercase)_ to "natuurlijk_persoon" _(lowercase)_

in the enum itself I can see

```java
public enum BetrokkeneType implements AbstractEnum<BetrokkeneType> {

    NATUURLIJK_PERSOON("natuurlijk_persoon"),

    NIET_NATUURLIJK_PERSOON("niet_natuurlijk_persoon"),

    VESTIGING("vestiging"),

    ORGANISATORISCHE_EENHEID("organisatorische_eenheid"),

    MEDEWERKER("medewerker");
 ```
 
 but in the generated type interface this translates to
 
 ```typscript
 /** @enum {string} */
        BetrokkeneType: "NATUURLIJK_PERSOON" | "NIET_NATUURLIJK_PERSOON" | "VESTIGING" | "ORGANISATORISCHE_EENHEID" | "MEDEWERKER";
```

I am unsure why. I also checked some other java enums and they also declare the keys in uppercase but the values in lowercase 🤔

Reply:
> I also checked some other java enums and they also declare the keys in uppercase but the values in lowercase 

Correct and most of these are generated code from the various OpenAPI specs and we have no control over these (that I know of).

Reply:
The enums used in / by ZAC are very messy and it is easy to break stuff. Also JSON (de)serialization can be tricky. Be warned.. ;-)

Reply:
I also suspect the JSON serialisation to handle the _UPPERCASE_ from the frontend to the _lowercase_ values in the backend. I do not want to touch the enum values themselves, I just wondered where the difference comes from and therefore a bit cautious to merge this.

Reply:
there is indeed an issue with the (de)serialization which is also caught by one of our tests

```
io.kotest.assertions.AssertionFailedError: expected:net.atos.client.zgw.zrc.model.BetrokkeneType<NATUURLIJK_PERSOON> but was:kotlin.String<"natuurlijk_persoon">
```

I will close this PR and add a story to the backlog to have a proper look at (de)serialization for enum values.

Reply:
ok! Good luck with those enums. :-)